appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.controller.js

```diff
@@ -283,6 +272,20 @@ export const postCheckAndConfirm = async (request, response) => {
 			)
 		);
 
+		if (request.session.reviewOutcome === 'complete') {
```

Comment:
This could be refactored to ```addNotificationBannerToSession({
	session: request.session,
	bannerDefinitionKey: request.session.reviewOutcome === 'complete' ? 'lpaqReviewComplete' : 'lpaqReviewIncomplete',
	appealId: currentAppeal.appealId
});```